library/core/src/primitive_docs.rs

```diff
@@ -1701,12 +1701,3 @@ impl<Ret, T> Clone for fn(T) -> Ret {
         loop {}
     }
 }
-
-// Fake impl that's only really used for docs.
-#[cfg(doc)]
-#[stable(feature = "rust1", since = "1.0.0")]
-#[doc(fake_variadic)]
-/// This trait is implemented on function pointers with any number of arguments.
-impl<Ret, T> Copy for fn(T) -> Ret {
-    // empty
-}
```

Comment:
This was added in #98180 by @notriddle but does not appear to affect any tests (anymore?). If this has a negative effect on docs, we should add tests for it and figure out how to get it back without having to add impls that don't typeck.